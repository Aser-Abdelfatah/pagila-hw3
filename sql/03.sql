/*
 * List the total amount of money that customers from each country have payed.
 * Order the results from most to least money.
 */
