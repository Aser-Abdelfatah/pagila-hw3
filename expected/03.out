                country                | total_payments 
---------------------------------------+----------------
 Afghanistan                           |          67.82
 American Samoa                        |          71.80
 Lithuania                             |          73.76
 Tonga                                 |          73.82
 Tunisia                               |          78.77
 Slovakia                              |          89.74
 Ethiopia                              |          91.77
 New Zealand                           |          92.76
 Madagascar                            |          93.78
 Saint Vincent and the Grenadines      |          96.75
 Senegal                               |         100.75
 Finland                               |         101.74
 French Guiana                         |         103.78
 Anguilla                              |         106.65
 Hungary                               |         111.71
 Iraq                                  |         111.73
 Kuwait                                |         111.74
 Bahrain                               |         112.75
 Brunei                                |         113.65
 North Korea                           |         113.69
 Liechtenstein                         |         114.72
 Faroe Islands                         |         114.72
 Estonia                               |         115.70
 Sri Lanka                             |         116.70
 Nepal                                 |         116.78
 Armenia                               |         118.75
 Tuvalu                                |         120.74
 Virgin Islands, U.S.                  |         122.68
 Malawi                                |         123.72
 Moldova                               |         127.66
 Gambia                                |         129.70
 Czech Republic                        |         133.71
 Zambia                                |         134.67
 Chad                                  |         135.68
 Turkmenistan                          |         136.73
 Greenland                             |         137.66
 Hong Kong                             |         142.70
 Sweden                                |         144.66
 Nauru                                 |         148.69
 Holy See (Vatican City State)         |         152.66
 Bolivia                               |         183.53
 Oman                                  |         187.50
 Cambodia                              |         191.47
 Myanmar                               |         194.48
 Kazakstan                             |         198.48
 Cameroon                              |         200.46
 Bulgaria                              |         204.50
 Congo, The Democratic Republic of the |         212.50
 Angola                                |         215.48
 Runion                                |         216.54
 Sudan                                 |         227.46
 Greece                                |         232.46
 Romania                               |         235.38
 French Polynesia                      |         235.46
 Azerbaijan                            |         245.43
 Switzerland                           |         252.39
 Kenya                                 |         253.46
 Puerto Rico                           |         254.39
 Yugoslavia                            |         259.43
 Latvia                                |         262.40
 Paraguay                              |         275.38
 Belarus                               |         277.34
 Austria                               |         307.22
 Morocco                               |         307.29
 Dominican Republic                    |         318.23
 Chile                                 |         328.29
 United Arab Emirates                  |         333.16
 Tanzania                              |         341.17
 Mozambique                            |         344.20
 Malaysia                              |         369.15
 France                                |         374.04
 Algeria                               |         383.10
 Ecuador                               |         390.13
 Bangladesh                            |         402.05
 Thailand                              |         419.04
 Israel                                |         422.01
 Peru                                  |         468.88
 Yemen                                 |         510.83
 Saudi Arabia                          |         523.79
 Pakistan                              |         525.72
 South Korea                           |         574.65
 Netherlands                           |         586.66
 Canada                                |         593.63
 Spain                                 |         606.58
 Venezuela                             |         683.30
 Egypt                                 |         694.39
 Colombia                              |         709.41
 Ukraine                               |         730.42
 Vietnam                               |         746.28
 Germany                               |         831.04
 Italy                                 |         831.11
 Poland                                |         877.97
 United Kingdom                        |         924.80
 Iran                                  |         950.75
 South Africa                          |        1204.15
 Taiwan                                |        1210.94
 Argentina                             |        1434.48
 Indonesia                             |        1510.33
 Nigeria                               |        1511.48
 Turkey                                |        1662.12
 Philippines                           |        2381.32
 Russian Federation                    |        3045.87
 Brazil                                |        3200.52
 Mexico                                |        3307.04
 Japan                                 |        3471.74
 United States                         |        4110.32
 China                                 |        5802.73
 India                                 |        6630.27
(108 rows)

